llvm/include/llvm/ADT/GenericUniformityImpl.h

```diff
@@ -342,6 +342,10 @@ template <typename ContextT> class GenericUniformityAnalysisImpl {
       typename SyncDependenceAnalysisT::DivergenceDescriptor;
   using BlockLabelMapT = typename SyncDependenceAnalysisT::BlockLabelMap;
 
+  // Use outside cycle with divergent exit
+  using UOCWDE =
```

Comment:
Use a less inscrutable type name?

Reply:
Just a suggestion, I would consider giving the name "TemporalDivergenceList" to the entire type ``SmallVector<std::tuple<InstructionT* ...``, and then derive other names from the public member types exposed by the SmallVector. In particular, the function that returns this vector as a range can benefit from simply saying ``auto`` as its return types. This instance of "really long derived names" is justification enough for just saying ``auto`` instead. There is no value in repeating all these names.

Reply:
Alternatively, UOCWDE can be renamed to ``TemporalDivergenceTuple``?